### DISCUSSION TITLE
GPU Compatbility

### DISCUSSION THREAD

**Discussion by jtcantin** [block-hczhai/block2-preview#112] [Q&A] [answered]
Hello Huanchen,

Does Block2 have any native GPU support, or would it only be though any dependencies that may have such support?

I looked at the docs, but did not see anything mentioned.

Thanks,
Josh

**Comment by hczhai** [comment, marked as answer]
There is currently no GPU support in block2, and it is unlikely that it can be supported through any dependencies.

**Reply by jtcantin** [reply]
Ok, thank you!
